Bottom: 79760b9da4d5c4f5e47763a79f668e6725bdf6e9
Top:    2a48d320de7c438e40d0bb2e8b080e61f78f7922
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-01 15:44:13 +0000

find.c: Move file-global "NumberOfPads" into the function which uses it


---

diff --git a/src/find.c b/src/find.c
index 9785bbe..c3bf707 100644
--- a/src/find.c
+++ b/src/find.c
@@ -273,7 +273,7 @@ static int thing_type;
 static bool User = false;    /* user action causing this */
 static bool drc = false;     /* whether to stop if finding something not found */
 static Cardinal drcerr_count;   /* count of drc errors */
-static Cardinal TotalP, TotalV, NumberOfPads[2];
+static Cardinal TotalP, TotalV;
 static ListType LineList[MAX_LAYER],    /* list of objects to */
   PolygonList[MAX_LAYER], ArcList[MAX_LAYER], PadList[2], RatList, PVList;
 
@@ -505,6 +505,7 @@ FreeComponentLookupMemory (void)
 static void
 InitComponentLookup (void)
 {
+  Cardinal NumberOfPads[2];
   Cardinal i;
 
   /* initialize pad data; start by counting the total number
